### PR TITLE
Fix bugs in `loss` module

### PR DESCRIPTION
**Pull request by bwohlberg**
Fix some bugs/oversights in initialization of linear operators.

### CODE REVIEW

**Review by Michael-T-McCann — CHANGES_REQUESTED** [review]
These changes look fine, but I don't see any changes to the `linop` module, which is at odds with the name of the PR. Is something missing here?

**Comment by bwohlberg** [comment]
> These changes look fine, but I don't see any changes to the `linop` module, which is at odds with the name of the PR. Is something missing here?

You're right. The change affected linop initialization, but was actually in the `loss` module. Title corrected accordingly.

File: scico/loss.py [hunk 1]
```diff
@@ -58,7 +58,8 @@ def __init__(
         Args:
             y: Measurement.
             A: Forward operator. Defaults to ``None``, in which case
-               `self.A` is a :class:`.Identity`.
+               ``self.A`` is a :class:`.Identity` with input shape
```
Comment (FernandoDavis) [inline scico/loss.py:61]: Backticks in self.A should be reverted I believe

**Review by FernandoDavis — APPROVED** [review]
LGTM

**Review by Michael-T-McCann — APPROVED** [review]

**Pull request merged**